src/components/flex/Flex.tsx

```diff
@@ -2,13 +2,16 @@ import React, { ReactNode } from 'react';
 import cx from 'classnames';
 import theme from './theme.css';
 
+const gaps = [0, 1, 2, 3, 4, 5, 6, 7, 8] as const;
+export type Gap = typeof gaps[number];
```

Comment:
Q: since `gaps` is not used, why not `export type Gap = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8;` directly?